tests/rptest/services/redpanda.py

```diff
@@ -447,14 +447,23 @@ def registered(self, node):
                     f"registered: peer {peer.name} admin API not yet available ({e})"
                 )
                 return False
+            found = None
```

Comment:
```
tests/redpanda: fixed registered method()
..
Fixed inconsitency in registered() method of redpanda ducktape service.
```

What is the inconsistency?

Reply:
i've added a comment to commit message. Basically the assert in that method was checking if all brokers returned by `/v1/brokers` are in metadata returned from Kafka API. This is incorrect since we should assert that all alive brokers are present in Metadata api response.